site2/docs/reference-cli-tools.md

```diff
@@ -328,8 +328,8 @@ Options
 
 |Flag|Description|Default|
 |---|---|---|
-|`-f`, `--files`|Comma-separated file paths to send; either -m or -f must be specified|[]|
-|`-m`, `--messages`|Comma-separated string of messages to send; either -m or -f must be specified|[]|
+|`-f`, `--files`|Comma separated file paths to send, either -m or -f or --null must be specified.|[]|
```

Comment:
Comma-separated file paths to send. At least one of the following values must be specified: `-m`, `-f` or `--null` .

Reply:
```suggestion
|`-f`, `--files`|Comma-separated file paths to send. At least one of the following values must be specified: `-m`, `-f` or `--null`.|[]|
```

Reply:
```suggestion
|`-f`, `--files`|Comma-separated file paths to send. At least one of the following values must be specified: `-m`, `-f` or `--null`.|[]|
```